ArcGISRuntimeSDKQt_CppSamples/Scenes/DisplayScenesInTabletopAR/DisplayScenesInTabletopAR.pro

```diff
@@ -0,0 +1,89 @@
+#-------------------------------------------------
+# Copyright 2019 Esri.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#-------------------------------------------------
+
+mac {
+    cache()
+}
+
+#-------------------------------------------------------------------------------
+
+CONFIG += c++14
+
+# additional modules are pulled in via arcgisruntime.pri
+QT += opengl qml quick
+
+TEMPLATE = app
+TARGET = DisplayScenesInTabletopAR
+
+ARCGIS_RUNTIME_VERSION = 100.7
+include($$PWD/arcgisruntime.pri)
+
+#-------------------------------------------------------------------------------
+
+HEADERS += \
+    DisplayScenesInTabletopAR.h
+
+SOURCES += \
+    main.cpp \
+    DisplayScenesInTabletopAR.cpp
+
+RESOURCES += DisplayScenesInTabletopAR.qrc
+
+#-------------------------------------------------------------------------------
+# AR configuration
+
+AR_TOOLKIT_SOURCE_PATH = /Users/jare8800/Applications/arvr # must be set to the path to toolkit sources
```

Comment:
might be better to leave this empty so the below qmake error will let the user know they need to set this. I'd keep the comment though